### PR TITLE
Ops/eject elicitation

### PR DESCRIPTION
**Pull request by kzalite**

### CODE REVIEW

File: app/ts/PreferencesTab/Preferences/Preferences.tsx [hunk 1]
```diff
@@ -17,71 +17,89 @@ import ScenarioButtons from './ScenarioButtons/ScenarioButtons';
...
+          <ScenarioSelection
+            scenarios={scenarios}
+            currentScenario={currentScenario}
+          />
+          <ScenarioButtons />
+          <PartialValueFunctions />
+          <PreferencesWeights />
+          <TradeOff />
+        </Grid>
+      );
```
Comment (chinook25) [inline app/ts/PreferencesTab/Preferences/Preferences.tsx:76]: extract component?

File: webpack.common.js [hunk 2]
```diff
@@ -31,7 +31,8 @@ let config = {
       {
         test: /\.ts(x?)$/,
         use: 'ts-loader',
-        exclude: [/frontend-test/, /node_modules/, /node-backend/, /tscomp/]
+        // exclude: [/frontend-test/, /node_modules/, /node-backend/, /tscomp/],
```
Comment (kzalite) [inline webpack.common.js:34]: remove comment

File: app/ts/PreferencesTab/Preferences/Preferences.tsx [hunk 3]
```diff
@@ -17,71 +17,89 @@ import ScenarioButtons from './ScenarioButtons/ScenarioButtons';
...
+          <Grid container justify="center" component={Box} mt={2}>
+            <AdvancedPartialValueFunction />
+          </Grid>
+        </AdvancedPartialValueFunctionContextProviderComponent>
+      );
+    } else if (
+      activeView === 'precise' ||
+      activeView === 'imprecise' ||
+      activeView === 'matching' ||
+      activeView === 'ranking'
```
Comment (chinook25) [inline app/ts/PreferencesTab/Preferences/Preferences.tsx:89]: isElicitationView(activeview)?

File: webpack.common.js (hunk 2 above)
Comment (chinook25) [inline webpack.common.js:34]: comments can be deleted?

**Review by chinook25 — APPROVED** [review]

**Review by DanielReid — APPROVED** [review]
small things

**Pull request merged**